src/app/checkout/checkout.component.js

```diff
@@ -63,6 +64,13 @@ class CheckoutController{
     });
   }
 
+  signedOut( event ) {
```

Comment:
The signOut code is only necessary on root level components that care about doing something different from the default that the nav.component provides. I wanted to keep the separation between sessionService and and the components.